js/Navigator.js

```diff
@@ -21,173 +21,172 @@ import Feedback from './tabs/service/Feedback';
 import Settings from './tabs/service/Settings';
 
 const stackHeaderConfig = {
-  headerBackTitle: 'Zurück',
-  headerTintColor: 'white',
-  headerStyle: {
-    backgroundColor: Colors.dhbwRed,
-    borderBottomWidth: 0,
-    ...Platform.select({
-      android: {
-        elevation: 0,
-        paddingTop: StatusBar.currentHeight
```

Comment:
Danke, habe diese spezielle Änderung in einem separaten Commit übernommen: https://github.com/DHBWLoerrach/CampusApp/commit/02e3412db250b8e6ea5c91aa8c79408039ede07c